apps/openmw/mwmechanics/aicombat.cpp

```diff
@@ -468,7 +468,16 @@ std::string chooseBestAttack(const ESM::Weapon* weapon)
         else
             attackType = "chop";
     }
-
+    else
+    {
+        float random = Misc::Rng::rollProbability();
+        if (random >= 2/3.f)
+        attackType = "thrust";
+        else if (random >= 1/3.f)
+        attackType = "slash";
+        else
+        attackType = "chop";
+    }
```

Comment:
The better option would be to rewrite the method `CharacterController::setAttackTypeRandomly()` as static and use that here and in the `CharacterController::updateWeaponState` to avoid code duplication. Character.hpp is already included in AiCombat.